### PR TITLE
Update Chrome beforeunload dialog control behavior

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

Chrome 119 includes an update to the [`beforeunload` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event) — it now triggers the "leave page?" confirmation dialog when `e.preventDefault()` is called upon the event firing, and doesn't do that when `e.returnValue` is set to an empty string. This PR updates the compat data appropriately.

See https://chromestatus.com/feature/4968823574233088 for the implementation details; the engineer has informed me that the actual release version will be 119).

This is great because Chrome now behaves like other browsers, so this event is immediately more useful than it was.

I intend to update the corresponding docs too.

### CODE REVIEW

File: api/BeforeUnloadEvent.json [hunk 1]
```diff
@@ -44,9 +44,17 @@
...
-            "chrome": {
-              "version_added": "30"
-            },
+            "chrome": [
+              {
+                "version_added": "119"
+              },
+              {
+                "version_added": "30",
+                "version_removed": "118",
```
Comment (queengooborg) [inline api/BeforeUnloadEvent.json:53]:
```suggestion
                "version_removed": "119",
```

File: api/BeforeUnloadEvent.json [hunk 2]
```diff
@@ -44,9 +44,17 @@
...
+                "partial_implementation": true,
+                "notes": "Incorrectly activated the dialog with an empty string value."
```
Comment (queengooborg) [inline api/BeforeUnloadEvent.json:55]:
```suggestion
                "notes": "Before Chrome 119, an empty string incorrectly activated the confirmation dialog."
```

**Review by queengooborg — CHANGES_REQUESTED** [review]

**Comment by queengooborg** [comment]
This was not ready to be merged and the descriptions don't have the proper HTML escapes, so it will cause rendering issues.

**Comment by chrisdavidmills** [comment]
@queengooborg dammnit, sorry about that. I was getting anxious because the corresponding content update got merged without the BCD being in place.

Do you need me to submit this again?

**Pull request merged**